src/demo/DemoTestSavingsService.java

```diff
@@ -195,6 +196,38 @@ public static void main(String[] args) {
 
 	}
 
+	/* Tests starting maturity on a saving account */
+	private static void testStartMaturityForSavingAccount() throws MambuApiException {
+
+		methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println("\nIn " + methodName);
+
+		String accountId = NEW_ACCOUNT_ID;
+
+		SavingsType savingAccountType = demoSavingsProduct.getProductType();
+		SavingsService savingService = MambuAPIFactory.getSavingsService();
+		AccountState accountState = savingService.getSavingsAccount(accountId).getAccountState();
+
+		if ((SavingsType.FIXED_DEPOSIT.equals(savingAccountType) || (SavingsType.SAVINGS_PLAN.equals(savingAccountType)))
+				&& (AccountState.ACTIVE.equals(accountState) || AccountState.DORMANT.equals(accountState))) {
+
+			SavingsService service = MambuAPIFactory.getSavingsService();
```

Comment:
We already have an instance of SavingsService at the line 208. Can't we use that one instead of creating a brand new one?